xwiki-platform-core/xwiki-platform-oldcore/src/main/resources/ApplicationResources.properties

```diff
@@ -2996,7 +2996,7 @@ xe.monitor.requests.size=Active requests size:
 xe.monitor.requests.page=Page:
 xe.monitor.thread=Thread:
 xe.monitor.requests.unfinished=Latest unfinished requests
-xe.monitor.requests.unfinished.description=These are requests that didn't reach "endRequest", but where cleaned-up by a reuse of threads. Only 32 requests max are kept in memory.
+xe.monitor.requests.unfinished.description=These are requests that didn't reach "endRequest", but where cleaned-up by a reuse of threads. Maximum 32 requests are kept in memory.
```

Comment:
I don't think this is correct. I think you need to write "A maximum of 32 requests are kept in memory.".

Reply:
I agree with you, but I also consider the currently proposed version an improvement from what already existed. It is a bit more concise, even if wrt. English correctness it's not the best (but then neither was the previous version).

Reply:
Yes changing an incorrect version with another incorrect one doesn't make us progress... ;)